aries-site/src/data/structure.js

```diff
@@ -403,7 +403,7 @@ export const structure = [
       'Need a place to host images in the cloud for use by your HPE services and apps?',
     preview: {
       image: {
```

Comment:
![Screen Shot 2020-04-20 at 8 14 48 PM](https://user-images.githubusercontent.com/42451602/79918975-b4f89f80-83ea-11ea-98ac-0a89112aaa66.png)
 Nice catch! I saw this but wasnt sure if there was a way around it